app/Http/Requests/ProviderRequest.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class ProviderRequest extends FormRequest
+{
+    public function rules(): array
+    {
+        return [
+            "providers" => [],
```

Comment:
Empty array?

Reply:
$request->providers return array of Providers ids ``//[1, 3, 9, 14]``

Should i change it to:

```php
"providers" => ["array"],
```
or remove that Request and use it in Controller just like:

```php
$request->providers
```
?

Reply:
array should be ok

Reply:
All done.